ui/app/components/clients/dashboard.js

```diff
@@ -121,14 +137,14 @@ export default class Dashboard extends Component {
     // clicked "Edit" Billing start month in Dashboard which opens a modal.
     if (dateType === 'startTime') {
       let monthIndex = this.arrayOfMonths.indexOf(month);
-      this.startTimeRequested = `${monthIndex + 1},${year}`; // "1, 2021"
+      this.startTimeRequested = [year.toString(), monthIndex]; // ['2021', 0] (e.g. January 2021) // TODO CHANGE TO ARRAY
       this.endTimeRequested = null;
     }
     // clicked "Custom End Month" from the calendar-widget
     if (dateType === 'endTime') {
       // use the currently selected startTime for your startTimeRequested.
       this.startTimeRequested = this.startTimeFromResponse;
-      this.endTimeRequested = `${month},${year}`; // "1, 2021"
+      this.endTimeRequested = [year.toString(), month];
```

Comment:
`month` here is a string if I'm reading correctly, do you mean `monthIndex`?

Reply:
endTime and startTime come in a little different. One comes in as a month name (startTime) whereas endTime comes in as an index because it's a calendar of months instead of a list. I'll add a comment in the code.